float32 kLST # linear stiffness
float32 kAST # angular stiffness
float32 kVST # volume stiffness
uint8 bending_distance
int32 flags # Default?
